website/docs/predeployed.md

```diff
@@ -26,12 +26,25 @@ If you want to deploy an instance of an account contract class not predeclared o
 
 The information on predeployed contracts is logged on Devnet startup. Predeployed accounts can be retrieved in JSON format by sending a `GET` request to `/predeployed_accounts`. With the additional query parameter `with_balance=true`, WEI and FRI balances at the pending state will be provided:
 
+```
+GET /predeployed_accounts
+```
+
```

Comment:
The optionality of parameters is already indicated by putting them in [square brackets]

Reply:
a5ca37e354fd258efbd7fa217b9c5ac2676f143d